tests/spec/jasmine-event-recur.js

```diff
@@ -207,6 +207,13 @@ describe("The Calendar Interval", function() {
         expect(recurrence.matches( moment(startDate).date(27) )).toBe(false);
     });
 
+    it("weeksOfMonthByDay should work", function() {
```

Comment:
If you have more than one test, you can have another describe in there.  It makes it much cleaner in karma runs to see what the problematic component is.